salt/modules/boto_route53.py

```diff
@@ -772,21 +772,21 @@ def update_record(
 
         salt myminion boto_route53.modify_record test.example.org 1.1.1.1 example.org A
 
-   retry_on_errors
-        Continue to query if the zone exists after an error is
-        raised. The previously used argument `retry_on_rate_limit`
-        was deprecated for this argument. Users can still use
-        `retry_on_rate_limit` to ensure backwards compatibility,
-        but please migrate to using the favored `retry_on_errors`
-        argument instead.
-
-    error_retries
-        Number of times to attempt to query if the zone exists.
-        The previously used argument `rate_limit_retries` was
-        deprecated for this arguments. Users can still use
-        `rate_limit_retries` to ensure backwards compatibility,
-        but please migrate to using the favored `error_retries`
-        argument instead.
+    retry_on_errors
+         Continue to query if the zone exists after an error is
+         raised. The previously used argument `retry_on_rate_limit`
+         was deprecated for this argument. Users can still use
+         `retry_on_rate_limit` to ensure backwards compatibility,
+         but please migrate to using the favored `retry_on_errors`
+         argument instead.
+
+     error_retries
```

Comment:
I don't think this part of the doc should have been shifted. It seems to shift some of these things correctly in these docs and some of them not.